views/UserProfile.html

```diff
@@ -59,237 +33,214 @@
+											<div class="dialog-title">Change Time</div>
+											<div class="dialog-content">
+												<input data-role="timepicker">
+											</div>
+											<div class="dialog-actions">
+												<button class="button js-dialog-close">Ok</button>
+												<button class="button js-dialog-close">Cancel</button>
+											</div>
+										</div>
+										<button onclick="Metro.dialog.open('#demoDialog1')" type="button" name="button" class="button info outline"> Change Time <span class="mif-alarm"></span></button>
 									</div>
-                                	<button onclick="Metro.dialog.open('#demoDialog2')" type="button" name="button" class="button info outline"> Change Address <span class="mif-location-city"></span></button>
-                                </div>
-                                <div class="pr-2">
-                                	<button type="button" name="button" class="button info outline"> Track Bus Location <span class="mif-location"></span></button>
-                                </div>
-                        	</div>
-                    	</div>
-                	</div>
-           		 </div>
-           	</div>
-        </div>
-
-
-        <br><br>
-
-
-        <div class="row">
-            <div class="col-xs-12">
-                <div class="container my-card">
-                    <div class="row">
-                        <div class="col-xs-12 col-md-9">
-                            <div class="row">
-                                <div class="col-md-12">
-                                        <h2>Trip 3</h2>
-                                </div>
-                            </div>
-                            <div class="row">
-                                <div class="col-md-12 text-muted">
-                                    <p>Pickup Location: ABC <br> Drop Location: XYZ <br> Time of Pickup: 8:00 AM </p>
-                                </div>
-                            </div>
-                            <br>
-                             <div class="row">
-                             	<div class="pr-2">
-                                	<button onclick="CancelTrip()" type="button" name="button" class="button info outline"> Cancel Trip <i class="mif-cancel"></i></button>
-                            	</div>
-                            	<div class="pr-2">
-                            		<div class="dialog" data-role="dialog" id="demoDialog1">
-									    <div class="dialog-title">Change Time</div>
-									    <div class="dialog-content">
-									        <input data-role="timepicker">
-									    </div>
-									    <div class="dialog-actions">
-									        <button class="button js-dialog-close">Ok</button>
-									        <button class="button js-dialog-close">Cancel</button>
-									    </div>
+									<div class="pr-2">
+										<div class="dialog" data-role="dialog" id="demoDialog2">
+											<div class="dialog-title">Change Address</div>
```

Comment:
There is an issue here that is to be resolved
"OK" and "Cancel" button are not seen

Reply:
They are seen at the bottom of the page. @GaganCJ can you look into the change address, because it is not rendering map due to 'L not defined' error.

Reply:
move the leaflet script to head tag, not at the end